modules/osdk-multi-arch-node-affinity.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-multi-arch-support.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-multi-arch-node-affinity_{context}"]
+= Safe scheduling an Operator's node affinity for multiple architecture compute nodes
+
+Node affinity is a set of rules used by the scheduler to define a pod's placement. Set node affinity requirements to ensure that your Operator's workloads only schedule to compute nodes with architectures that are compatible with the images in the pod. 
```

Comment:
Here is probably where I would add a comment about how node affinity is important to set when your operator is expected to work in a cluster with compute nodes of multiple architectures. You can learn more about multi-arch compute clusters in *doc link*.